app/database/dataStore.go

```diff
@@ -2,6 +2,8 @@ package database
 
 import (
 	"math/rand"
+
+	t "github.com/France-ioi/AlgoreaBackend/app/types"
```

Comment:
I would recommend not to alias this and just use `types`. In Go, package is a part of the name of whatever is exposed there.